production-config/README.md

```diff
@@ -0,0 +1,87 @@
+## Deploy OpenFisca-Web-API in production
+
+> This how-to explains how to host OpenFisca-Web-API in *production*, for example to serve `api.xxx.com`.
+> If you just need to develop with a local install of OpenFisca-Web-API, you need to read
+> the main [README](https://github.com/openfisca/openfisca-web-api/blob/master/README.md).
+
+The following instructions assume you use the Debian Jessie GNU/Linux distribution and that Python 2.7 is installed.
+
+Here is the big picture:
+- Use OpenFisca-Web-API Python package published to PyPI; do not clone this git repository.
+- [Nginx](https://www.nginx.com/) serves content on the port 80 via a virtual host, and proxyfies a subdomain to
+  [gunicorn](http://gunicorn.org).
+- The application is installed in a Python virtualenv.
+  - We recommend using [pew](https://github.com/berdario/pew) because it's written in Python so it's shell agnostic.
+- For the sake of the example we're going to make the Web API serve the OpenFisca-France country.
+
+-  web server
+-  WSGI-compatible Python web server
```

Comment:
There seems to be a small markup glitch here.

![capture d ecran 2017-03-03 a 19 42 39](https://cloud.githubusercontent.com/assets/222463/23564109/9d4c3842-0049-11e7-9829-7c518c796b03.png)